microsoft-edge/dualengine/reference/index.md

```diff
@@ -0,0 +1,35 @@
+---
+description: DualEngine Win32 C++ Reference
+title: DualEngine Win32 C++ Reference
+author: MSEdgeTeam
+ms.author: edgededev
+ms.prod: microsoft-edge
+ms.date: 11/16/2023
+keywords: dual engine, dualengine, iemode, win32 apps, win32, edge, ie mode
+---
+# Reference (DualEngine Win32 C++)
+> [!IMPORTANT]
+> The DualEngine API is a limited access feature. Contact dualengineapiaccess@microsoft.com for more information.
+
+The Microsoft DualEngine API enables you to host and control an Internet Explorer in your application.  For more information, see [Overview of Microsoft DualEngine API](../index.md) and [Getting Started with DualEngine](../get-started.md). 

```

Comment:
add qualifiers "Win32" and "C++" in this first sentence, instead of in the top-of-page title